### PR TITLE
Cme match error

### PR DESCRIPTION
**Pull request by jlikhuva**
@caesar2164 @caseylitton

### CODE REVIEW

**Comment by caseylitton** [comment]
+1

**Comment by caseylitton** [comment]
Also your PR message would be better as
"
@caesar2164 @caseylitton PR for {description of fix}

{Description of how to turn on CME registration for devstack}

{link to trello card}
"

When there are specific lms or cms envs needed it's nice to add them to the message so other people can quickly test.

**Comment by caseylitton** [comment]
Giulio just showed me the steps so... 

On devstack in lms.env.json

Add "openedx.stanford.djangoapps.register_cme" to ADDL_INSTALLED_APPS

Create "REGISTRATION_EXTENSION_FORM": "openedx.stanford.djangoapps.register_cme.forms.ExtraInfoForm", above REGISTRATION_EXTRA_FIELDS

Make all 'optional' values in REGISTRATION_EXTRA_FIELDS be 'hidden'

ENABLE_COMBINED_LOGIN_REGISTRATION needs to be true

USE_CUSTOM_THEME needs to be true

Clone cme-theme renamed to cme in /edx/app/edxapp/themes/

File: openedx/stanford/djangoapps/register_cme/forms.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ class Meta(object):
                 'required': u'Please enter your Postal Code.',
             },
             'professional_designation': {
-                'required': u'Please select your Professional Designation.',
+                'required': u'Please select your Medical Designation or Professional Title.',
```
Comment (dcadams) [inline openedx/stanford/djangoapps/register_cme/forms.py]: If it is no longer Professional Designation then we should rename all instances in the code.

**Comment by caesar2164** [comment]
https://stackoverflow.com/questions/3042437/change-commit-author-at-one-specific-commit

File: openedx/stanford/djangoapps/register_cme/forms.py (hunk 1 above)
Comment (caseylitton) [inline openedx/stanford/djangoapps/register_cme/forms.py]: The refactor will be his next PR so we can get this PR merged.

**Comment by potsui** [comment]
Revised Steps for switching to CME:

1. On devstack in lms.env.json (sudo if read-only in vim)
2. Add "openedx.stanford.djangoapps.register_cme" to ADDL_INSTALLED_APPS
3. Create "REGISTRATION_EXTENSION_FORM": "openedx.stanford.djangoapps.register_cme.forms.ExtraInfoForm", above REGISTRATION_EXTRA_FIELDS
4. Make all 'optional' values in REGISTRATION_EXTRA_FIELDS be 'hidden' (note: upon reset these are: gender, goals, level_of_education, mailing_address, and year_of_birth)
5. ENABLE_COMBINED_LOGIN_REGISTRATION needs to be true
6. USE_CUSTOM_THEME needs to be true
7. Change "THEME_NAME" to "cme"
8. Clone cme-theme renamed to cme in /edx/app/edxapp/themes/

**Pull request merged**